core/App/ehsm_marshal.h

```diff
@@ -77,4 +77,4 @@ ehsm_status_t marshal_msg3_to_json(sgx_ra_msg3_t *p_msg3, RetJsonObj *retJsonObj
  */
 ehsm_status_t unmarshal_att_result_msg_from_json(std::string ra_att_result_msg, sample_ra_att_result_msg_t *req_att_result_msg);
```

Comment:
update the commit title as
"
Refactor the crypto interfaces to leverage sgx-ssl

Intel-sgx-ssl is an opensource project that based on the underlying OpenSSL version 1.1.1q, providing a full-strength general purpose cryptography library. 
https://github.com/intel/intel-sgx-ssl

In this patch, we will refine the framework of the core enclave and leverage the sgx-ssl to provide more algorithms such as: aes(gcm-128, gcm-192, gcm-256), rsa(2048, 3072, 4096), ecc(p224, p256, p384, p521), sm2/sm4.

Besides, all the test cases have been updated and add more test cases for each algorithms.

Test: passed all the unit test.

Signed-off-by: Yan, Shaopu <shaopu.yan@intel.com>
Signed-off-by: Yang Huang <yang.huang@intel.com>
Signed-off-by: wanghouqi <houqix.wang@intel.com>
"